assets/tests/components/mapPage.test.tsx

```diff
@@ -90,6 +97,31 @@ describe("MapPage", () => {
     expect(asFragment()).toMatchSnapshot()
   })
 
+  test("renders stations on zoom", async () => {
+    ;(useStations as jest.Mock).mockImplementationOnce(() => [
+      {
+        id: "station-1",
+        name: "Station 1",
+        locationType: LocationType.Station,
+        lat: 42,
+        lon: -71,
+      },
+    ])
+
+    const { container } = render(
+      <StateDispatchProvider state={initialState} dispatch={jest.fn()}>
+        <BrowserRouter>
+          <MapPage />
+        </BrowserRouter>
+      </StateDispatchProvider>
+    )
+
+    await userEvent.click(screen.getByRole("button", { name: "Zoom in" }))
+    await userEvent.click(screen.getByRole("button", { name: "Zoom in" }))
+
+    expect(container.innerHTML).toContain("m-station-icon")
```

Comment:
suggestion: how do you feel about using this instead?

```suggestion
    expect(container.querySelector(".m-station-icon")).toBeVisible()
```